build.gradle

```diff
@@ -687,6 +710,11 @@ task sendCoverageToCodacy(type: JavaExec, dependsOn: jacocoTestReport) {
     ]
 }
 
+cucumberReports {
```

Comment:
some explanatory comments here would be good too. I know we don't have many comments in the build.gradle but we should start getting in the habit to add them other wise it's difficult to understand why a piece of config is there